Makefile

```diff
@@ -207,6 +207,10 @@ push-mmf-go-mmlogic-simple-image: build-mmf-go-mmlogic-simple-image
 	docker push $(REGISTRY)/openmatch-mmf-go-mmlogic-simple:$(TAG)
 	docker push $(REGISTRY)/openmatch-mmf-go-mmlogic-simple:$(ALTERNATE_TAG)
 
+push-mmf-go-grpc-serving-simple-image: build-mmf-go-grpc-serving-simple-image
```

Comment:
cloudbuild.yaml is missing this as an image artifact.

Reply:
Added now.